indico/modules/events/contributions/util_test.py

```diff
@@ -125,3 +125,6 @@ def test_import_contributions_errors(db, dummy_event):
 
     e = _check_importer_exception(dummy_event, b'2010-02-23T00:00:00,15min,Test,,,,')
     assert 'parse duration' in e.message
+
+    e = _check_importer_exception(dummy_event, b'2010-02-23T00:00:00,15min,Test,,,,foobar')
```

Comment:
this still contains an invalid duration so the test might fail depending on the order of the checks. better take an otherwise valid string (except for the bad email of course)